src/_utils/branding.ts

```diff
@@ -167,6 +168,22 @@ export const color: Color = {
   tapHighlight: 'rgba(221, 221, 221, .4)', // gray, 40%
 }
 
+
+export enum ResponsiveMode {
+  SMALL = 'small',
+  LARGE = 'large',
+}
+
+export const computeResponsiveModeClass = (mode:ResponsiveMode) => {
```

Comment:
Why not having the CSS breakpoint in Kirk to avoid passing responsive flag via JavaScript ?

Reply:
CSS media query breakpoints are related to the viewport, not the width of a particular layout element. To support future multi-column layouts, we need to make sure that the component is responsive to its containing layout column, not the viewport size.
But maybe, we don't care for now ? and I go for viewport based media queries that will break when we introduce multi-column desktop layouts ? That will simplify this PR a bit and the integration points in kairos.

We will have the same discussion about layout paddings/margins in future PRs (e.g next week probably).

Reply:
So you could have component with the responsive mode large and small on the same page depending on their container ?

Reply:
yes.
As discussed offline, I will remvoe for now the responsive mode and add a responsive breakpoint in TheVoice. We will iterate on that when we have actual desktop specs.